### PR TITLE
Fix several typos, and rephrase one whole sentence

### PR DESCRIPTION
**Pull request by jafingerhut**

### CODE REVIEW

File: docs/v1/P4Runtime-Spec.mdk [hunk 1]
```diff
@@ -1730,7 +1731,7 @@ For example, the following P4~16~ objects involve complex types that need to be
 exposed in P4Runtime in order to support runtime operations on these objects.
 
 ~ Begin P4Example
-value_set<tuple<bit<16>, bit<8> > >(16) pvs_complex;
+value_set<tuple<bit<4>, bit<8> > >(16) pvs_complex;
```
Comment (chrispsommers) [inline docs/v1/P4Runtime-Spec.mdk]: Why the change to 4 bits?

Comment (jafingerhut) [reply docs/v1/P4Runtime-Spec.mdk]: This is really just a nit-pick change -- the field being compared against in the example is the IHL field of the IPv4 header, which is 4 bits wide in the standard.  This code snippet is short enough that it does not contain the explicit declaration of the IPv4 header type.  My OCD kicked in while reading it :-)

Comment (chrispsommers) [reply docs/v1/P4Runtime-Spec.mdk]: @jafingerhut Got it, I was wondering.  And in this case,  OCD stands for "Occupational Capability Demonstration." :)

Comment (jafingerhut) [reply docs/v1/P4Runtime-Spec.mdk]: Oh, and I was misremembering that it was the IPv4 IHL field, but it was another 4-bit field, the IPv4 version field.

File: docs/v1/P4Runtime-Spec.mdk [hunk 2]
```diff
@@ -4042,9 +4043,9 @@ the P4Info, the opaque target-dependent forwarding-pipeline configuration data
 to uniquely identify such configuration. See the [Forwarding-Pipeline
 Configuration](#sec-p4-fwd-pipe-config) section for details.
 
-A P4Runtime server running on an a typical device may not support
+A P4Runtime server running on an a non-programmable device may not support
```
Comment (antoninbas) [inline docs/v1/P4Runtime-Spec.mdk]: there is a typo that is not of your making on this line, there is an extra article ("an a")

File: docs/v1/P4Runtime-Spec.mdk [hunk 3]
```diff
@@ -4322,7 +4323,7 @@ caption: "P4Runtime Metadata Translation for the Portable Switch Architecture" }
 [psa-metadata-translation]: build/psa-metadata-translation.[svg,png] \
 { height: 7cm; page-align: here }
 
-Figure [#fig-psa-metadata-translation] above illustrates a motivating example,
+Figure [#fig-psa-metadata-translation] illustrates a motivating example,
```
Comment (antoninbas) [inline docs/v1/P4Runtime-Spec.mdk]: lol, nice catch

File: docs/v1/P4Runtime-Spec.mdk (hunk 2 above)
Comment (jafingerhut) [reply docs/v1/P4Runtime-Spec.mdk]: Done.
